ui/src/app/pages/workspace/workspace-about.tsx

```diff
@@ -233,6 +233,21 @@ export const WorkspaceAbout = fp.flow(
       }
     }
 
+    workspaceGcpBillingSpendUrl(disabled: boolean): string {
+      // Don't render Billing URl because it is not needed and user might not have permission to know the attached billing account.
```

Comment:
If the second part of this sentence is true, then the backend shouldn't be returning it..

Reply:
Sorry for the confusing. Turns out that the user does not need to have permission on a billing account at all to view the spend report. Make sense to let them to view it.  I will simply the method to remove the disabled param from the method because just for the 1st part part of this sentence is bit overkill...wdyt?

Reply:
yep, sounds good